<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>441</width>
       <height>138</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>40</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="b2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="b3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="b7">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>|</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="b1">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="b5">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="b4">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="b8">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>//</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="b6">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&amp;&amp;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>421</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QPushButton" name="num1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="num2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="num3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="num4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>421</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QPushButton" name="temp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="transfer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QLabel" name="expression">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>e</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="operator_show">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>temp</zorder>
     <zorder>expression</zorder>
     <zorder>clear</zorder>
     <zorder>operator_show</zorder>
     <zorder>transfer</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>options</string>
    </property>
    <widget class="QMenu" name="menudifficulty">
     <property name="title">
      <string>difficulty</string>
     </property>
     <addaction name="actioneasy"/>
     <addaction name="actionnormal"/>
     <addaction name="actionhard"/>
     <addaction name="actionlunatic"/>
    </widget>
    <addaction name="actionnew_game"/>
    <addaction name="actionhelp"/>
    <addaction name="menudifficulty"/>
    <addaction name="actioncheat"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actioneasy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>easy</string>
   </property>
  </action>
  <action name="actionnormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>normal</string>
   </property>
  </action>
  <action name="actionhard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>hard</string>
   </property>
  </action>
  <action name="actionlunatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>lunatic</string>
   </property>
  </action>
  <action name="actionnew_game">
   <property name="text">
    <string>new game</string>
   </property>
  </action>
  <action name="actioncheat">
   <property name="text">
    <string>cheat</string>
   </property>
  </action>
  <action name="actiondifficulty">
   <property name="text">
    <string>difficulty</string>
   </property>
  </action>
  <action name="actionshow_max_cheat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show max cheat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
